### PR TITLE
Add archive.pressthink.org and oxfordamerican.com

### PR DESCRIPTION
**Pull request by naamval**

### CODE REVIEW

File: archive.pressthink.org.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# Place this file inside your site_config/custom/ folder
+
+title: //h3[contains(concat(' ',normalize-space(@class),' '),' title ')]
+body: //div[contains(concat(' ',normalize-space(@class),' '),' blogbody ')]
+date: //div[contains(concat(' ',normalize-space(@class),' '),' date ')]
+
+strip: //h3[contains(concat(' ',normalize-space(@class),' '),' title ')]
+strip: //span[contains(concat(' ',normalize-space(@class),' '),' posted ')]
+
+test-url: http://archive.pressthink.org/2003/09/08/basics_master.html
```
Comment (j0k3r) [inline archive.pressthink.org.txt]: This must be `test_url` instead of `test-url`

**Comment by fivefilters** [comment]
Thanks! 👍  (And changed to test_url)

**Comment by j0k3r** [comment]
@fivefilters the second file also have the `test-url` problem :)

**Comment by naamval** [comment]
Oops! Thanks for the revision.

**Pull request merged**